### PR TITLE
Minor cleanup in QEMU CI

### PR DESCRIPTION
**Pull request by lukaszstolarczuk**
A little cleanup around QEMU job in CI. It contains a few commits, each can be reviewed separately. It's a small introduction to next changes in QEMU CI, I have in mind.

### CODE REVIEW

File: scripts/qemu/start_qemu.sh [hunk 1]
```diff
@@ -3,28 +3,25 @@
...
-set -x
 set -e
 
-config_file=$1
+# The config file name (should be located in ./configs/ sub-dir)
+config_name=$1
 
-python3 scripts/qemu/qemu_config.py $config_file
+# Parse the config file to get topology info and fix escaped single quotes
+parsed_config=$(python3 "$(dirname $0)/parse_config.py" "$(dirname $0)/configs/${config_name}" | sed s/''\''/'/g)
```
Comment (ldorau) [inline scripts/qemu/start_qemu.sh:12]: nit: maybe print out this `parsed_config` ?

File: .github/workflows/qemu.yml [hunk 2]
```diff
@@ -59,54 +61,46 @@ jobs:
...
-            CI_REPO="$GITHUB_REPOSITORY"
-        fi
+    - name: Build UMF in QEMU
+      run: |
+        umf/scripts/qemu/start_qemu.sh default.xml
 
-        scp -P 2222 ${{github.workspace}}/scripts/qemu/run-build.sh cxltest@127.0.0.1:/home/cxltest
-        scp -P 2222 ${{github.workspace}}/scripts/qemu/run-tests.sh cxltest@127.0.0.1:/home/cxltest
-        ssh cxltest@127.0.0.1 -p 2222 -t "bash /home/cxltest/run-build.sh https://github.com/$CI_REPO ${{env.CI_BRANCH}}"
+        rsync -az -e "ssh -p 2222" ${{github.workspace}}/umf/ testuser@127.0.0.1:/home/testuser/
```
Comment (ldorau) [inline .github/workflows/qemu.yml:85]: Could you add a comment what you copy here (using `rsync`)?

**Review by ldorau — APPROVED** [review]

File: .github/workflows/qemu.yml (hunk 2 above)
Comment (lukaszstolarczuk) [reply .github/workflows/qemu.yml:86]: Done.

File: scripts/qemu/start_qemu.sh (hunk 1 above)
Comment (lukaszstolarczuk) [reply scripts/qemu/start_qemu.sh:12]: the whole command, along with the parsed part, is printed, thanks to `set -x`

**Review by ldorau — APPROVED** [review]

File: scripts/qemu/parse_config.py [hunk 3]
```diff
@@ -50,6 +53,9 @@ def parse_topology_xml(tpg_file_name: str) -> TopologyCfg:
         result.check_returncode()
         libvirt_args = result.stdout.decode("utf-8").strip()
 
+        if verbose_mode != False:
```
Comment (PatKamin) [inline scripts/qemu/parse_config.py:56]: if verbose_mode:

Comment (lukaszstolarczuk) [reply scripts/qemu/parse_config.py:56]: I know this should work, but:
- I followed the same pattern as in other places here,
- plus, we just want to make sure it's any value, but not the default one (`os.getenv("ENABLE_VERBOSE", False)`)

If you want I can change it in all places, but I'd personally leave it as is.

**Review by PatKamin — APPROVED** [review]

**Pull request merged**